kernel/mem/paging.cpp

```diff
@@ -88,7 +94,7 @@ static void px_paging_init_dir() {
         px_map_kernel_page_table(i, &page_tables[i]);
         // clear out the page tables
         for (int j = 0; j < PAGE_ENTRIES; j++) {
-            page_tables[i].pages[j] = (px_page_table_entry_t){ 0 };
+            page_tables[i].pages[j] = (px_page_table_entry_t){ /* ZERO */ };
```

Comment:
I prefer the actual zero, but maybe you did this because of warnings?

Reply:
Yes, I did this because of the warnings. In C this would set everything to 0, but in C++ this only sets the first struct member to 0. (The rest are technically set to 0, but the compiler complains since the correct way is to leave the `{ }` blank.

Reply:
Ooo. I did not know that. I kinda don't like it, but I guess it is how it is. I'm also not a fan of the comment, but that's whatever.

Reply:
I can remove the comments, but I figured it would be better to have something explaining that it's all zero'd out.

Reply:
Ok